ui/app/AppLayouts/Wallet/controls/BuyCryptoProvidersDelegate.qml

```diff
@@ -0,0 +1,43 @@
+import QtQuick 2.15
+
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+StatusListItem {
+    id: root
+
+    property bool isUrlLoading: false
+    property bool urlsNeedParams: urlsNeedParameters
+
+    title: name
+    subTitle: description
```

Comment:
where is the `name`, `description` and `logoUrl` (below) coming from? You should define formal properties for these, and just pass them down from above